listenbrainz/webserver/views/entity_pages.py

```diff
@@ -338,3 +341,110 @@ def album_entity(release_group_mbid: str):
 @release_group_bp.get('/<path:path>/')
 def release_group_redirect(path):
     return render_template("index.html")
+
+
+@recording_bp.get("/",  defaults={'path': ''})
+@recording_bp.get('/<path:path>/')
+def recording_redirect(path):
+    return render_template("index.html")
+
+
+@track_bp.route("/",  defaults={'path': ''})
+@track_bp.get('/<recording_mbid>/')
```

Comment:
Is it expected to defined both .route and .get?

Reply:
It should be .get for both. Good catch